pandas/plotting/_core.py

```diff
@@ -637,7 +638,9 @@ class PlotAccessor(PandasObject):
     yticks : sequence
         Values to use for the yticks.
     xlim : 2-tuple/list
+        Set or query x-axis limits.
```

Comment:
I don't think this helps understand what is expected much. Let me know if you need help understanding better how to use these parameters.

Reply:
I'm gonna changed it to:
```
    xlim : 2-tuple/list
        Values to use for x-axis limits.
```
Do you think it helps enough?

Reply:
I would personally be more specific. I expect some users to not immediately understand what "x-axis limits" mean.

I think something like "min and max values displayed in the X axis" could help some readers understand this faster.

Reply:
Thanks, I'll work on it.